bin/makemessages.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+set -eu
+
+# generate initial .PO files from msgids in template and view files
+
+python manage.py makemessages -a --no-obsolete
+
+# put back bug-report link
+
+sed -i 's/Report-Msgid-Bugs-To\:/Report-Msgid-Bugs-To\: https\:\/\/github.com\/cal-itp\/benefits\/issues/g' benefits/locale/**/**/**.po
```

Comment:
My bash etc. is horrible, but doesn't a single `**` represent "any directory level?"

Also the file glob should be `*.po` no?

Reply:
So I can't find any explicit documentation for [sed](https://linux.die.net/man/1/sed) that backs this up, but from trying it out myself, the `input-file` parameter doesn't seem to follow glob pattern. You are correct that the second star is redundant.

✔️ Using `benefits/locale/*/*/*.po` gets the same result. 

❌ Using `benefits/locale/**/*.po` results in: 
```
calitp@10718e11bd6a:~/app$ ./bin/makemessages.sh 
ENABLE_AZURE_INSIGHTS:  False
ENABLE AZURE:  False
[22/Jul/2022 19:41:59] INFO benefits.oauth.client:42 Registering OAuth clients
processing locale en
processing locale es
sed: can't read benefits/locale/**/*.po: No such file or directory
``` 

I'll change it to the less redundant and less misleading pattern (`benefits/locale/*/*/*.po`).